templates/article/article_show.html.twig

```diff
@@ -1,26 +1,38 @@
 {% extends 'base.html.twig' %}
 
-{% block heading %}
-    {% if article %}
-        {{ article.name }}
-    {% else %}
-        Такой статьи не существует.
-    {% endif %}
+{% block page_heading %}
+    <div class="row justify-content-center" >
+        <div class="col-11">
+            <h2 class="sub-header text-center brand-txt">
+                {% block heading %}
+                    {% if article %}
+                        {{ article.name }}
+                    {% else %}
```

Comment:
@daria-sieroshtan 
I think you don't need this - user should get 404 in this case